src/modals/schema/EditSchema.vue

```diff
@@ -42,10 +47,10 @@ import { schemaStore, navigationStore } from '../../store/store.js'
 				@click="editSchema()">
 				<template #icon>
 					<NcLoadingIcon v-if="loading" :size="20" />
-					<ContentSaveOutline v-if="!loading && schemaStore.schemaItem?.id" :size="20" />
+					<ContentSaveOutline v-if="!loading && schemaStore.schemaItem?.id && !createAnother" :size="20" />
 					<Plus v-if="!loading && !schemaStore.schemaItem?.id" :size="20" />
```

Comment:
![image](https://github.com/user-attachments/assets/5dd4e03f-c484-470a-8f7d-5e3acd5cbec4)

If you Edit and existing schema and toggle "Create another" the icon will go away

Reply:
update:
On an edit schema you should not be able to create another
![image](https://github.com/user-attachments/assets/12b3480c-e216-444b-b199-bd3fe3696120)